aiida_vasp/io/vasprun.py

```diff
@@ -7,13 +7,58 @@
 import datetime as dt
 import numpy as np
 
+from aiida_vasp.io.parser import BaseFileParser
 
-class VasprunParser(object):
+DEFAULT_OPTIONS = {'quantities_to_parse': ['occupations', 'vrp_pdos', 'vrp_tdos']}
+
+
+class VasprunParser(BaseFileParser):
```

Comment:
Unit tests to aiida_vasp/io/tests/test_vasprun_io.py, please

Reply:
Added a test for the VasprunParser and also promoted 'lxml' to a required package